nemo/collections/nlp/models/language_modeling/lm_model.py

```diff
@@ -162,34 +162,31 @@ def validation_epoch_end(self, outputs):
         Called at the end of validation to aggregate outputs.
         :param outputs: list of individual outputs of each validation step.
```

Comment:
this is a wrong docstring format - our sphynx build won't be able to digest it. Please use the style like in other models. 
BTW, in PyCharm you can set the style

Reply:
Yes, I have a plugin in vscode that auto-generates. This was from the original implementation.

Reply:
Fixed.